features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -283,6 +284,34 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * The SlowFramesListener provides statistical data to help identify performance issues related to UI rendering:
+         *
+         * - slowFrames: A list of records containing the timestamp and duration of frames where users experience
+         *   jank frames within the given view.
+         *
+         * - slowFrameRate: The rate of slow frames encountered during the view's lifetime.
+         *
+         * - freezeRate: The rate of freeze occurrences during the view's lifetime.
+         *
+         *
+         *
+         * This configuration sets the parameters for the SlowFramesListener, which are used to calculate the slow frames array,
+         * slow frame ratio, and freeze ratio. For additional details, refer to [SlowFrameListenerConfiguration].
+         *
+         * Assigning a null value to this property will disable the SlowFramesListener and stop the computation of the
```

Comment:
```suggestion
         * Assigning a null value to this property will disable the [SlowFramesListener] and stop the computation of the
```

Also it is worth mentioning if it is enabled or disabled by default.